test-d/json.test-d.ts

```diff
@@ -0,0 +1,27 @@
+import { expectType } from 'tsd'
+import * as st from '../src'
+
+const data: unknown = null
+
+// basic
+{
```

Comment:
Added these. Not sure if there should be more / if there are other tests that would be preferred.

Reply:
Looks good to me.